build.gradle

```diff
@@ -183,25 +183,22 @@ File scriptFile(String relativePath) {
 
 def defaultScriptArgs = [mps_home, build_dir, artifacts_dir, ext.buildDate, ext.pluginVersion, ext.skipUnmodifiedModels]
 
+// properties for default configuration
```

Comment:
I think the property name here are self explanatory enough that we don't need such a comment that stats the obvious. I would suggest we remove it or replace with something more valuable for instance a link to the documentation of these properties.

Reply:
I've added the link to the mps-gradle-plugin docu to the comment.